src/components/Layout/SaasSelector.tsx

```diff
@@ -0,0 +1,84 @@
+import { useTranslation } from 'react-i18next'
+import SelectCustom, { SelectCustomProps } from '~components/OrganizationSaas/Dashboard/SelectCustom'
+
+export const OrganzationTypesSelector = ({ ...props }: Omit<SelectCustomProps, 'options'>) => {
```

Comment:
As all CustomTypesSelectors will have the same props type, you can declare a type for it:

```ts
export type CustomTypeSelectorProps =  Omit<SelectCustomProps, 'options'>
```

And then use it on the types selector you are declaring

```ts
export const OrganzationTypesSelector = ({ ...props }: CustomTypeSelectorProps) => {
```

Just to don't repeat the code and for scallability

Reply:
Doesn't matter